README.md

```diff
@@ -52,34 +60,49 @@ Options:
 
 ## annotated spec:
```

Comment:
This is technically not an _annotated spec_ since we aren't showing a contiguous TOML file anymore, perhaps rename the header to something 🤔 ?

Reply:
it's a good point 🤔